scripts/sponsoring/1_add-modules-to-proxy.ts

```diff
@@ -0,0 +1,87 @@
+import { BigNumber } from '@ethersproject/bignumber';
+import { BytesLike } from '@ethersproject/bytes';
+import { time } from '@nomicfoundation/hardhat-network-helpers';
+import hre, { ethers } from 'hardhat';
+import CONFIG from '../../config/config.json';
+import {
+    IexecPoco1Delegate__factory,
+    IexecPoco2Delegate__factory,
+    IexecPocoAccessorsDelegate__factory,
+    TimelockController__factory,
+} from '../../typechain';
+import { Ownable__factory } from '../../typechain/factories/rlc-faucet-contract/contracts';
+import {
+    encodeModuleProxyUpdate,
+    logTxData,
+    printBlockTime,
+    printFunctions,
+} from '../upgrades/upgrade-helper';
+
+(async () => {
+    const chainId = (await ethers.provider.getNetwork()).chainId;
+    const deploymentOptions = CONFIG.chains[chainId].v5;
+    console.log('Link Boost functions to proxy:');
+    const erc1538ProxyAddress = deploymentOptions.ERC1538Proxy;
+    const iexecPoco1DelegateAddress = (await hre.deployments.get('IexecPoco1Delegate')).address;
+    const iexecPoco2DelegateAddress = (await hre.deployments.get('IexecPoco2Delegate')).address;
+    const iexecPocoAccessorsDelegateAddress = (
+        await hre.deployments.get('IexecPocoAccessorsDelegate')
+    ).address;
+    const [account] = await hre.ethers.getSigners();
```

Comment:
```suggestion
    const [account] = await hre.ethers.getSigners();
```
Can we use a more meaningful name. anyone?